cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ArtifactHttpHandlerTest.java

```diff
@@ -162,6 +162,23 @@ public void testPluginInvalid() throws Exception {
                                           parents, pluginClasses).getStatusLine().getStatusCode());
   }
 
+  @Test
+  public void testEmptyPluginClasses() throws Exception {
+    // tests that the addArtifact when called with empty plugin classes gets deployed. This simulates the call made
+    // by UI
+    File wordCountArtifact = buildAppArtifact(WordCountApp.class, "wordcount.jar");
+    ArtifactId wordcountId1 = NamespaceId.DEFAULT.artifact("wordcount", "1.0.0");
+    Assert.assertEquals(HttpResponseStatus.OK.code(),
+                        addArtifactHelper(Id.Artifact.fromEntityId(wordcountId1),
+                                          Files.newInputStreamSupplier(wordCountArtifact),
+                                          Collections.emptySet(), "").getStatusLine().getStatusCode());
+
+    // test get /artifacts endpoint
```

Comment:
extra `/` ?

Reply:
The tests in this class are following a convention of <method> /<path> this is to be consistent with it.